src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestEntity.cs

```diff
@@ -10,10 +10,12 @@ public class AssetInterestEntity
         
         public decimal InterestRate { get; set; }
         
-        public DateTimeOffset CreatedAt { get; set; }
```

Comment:
it should be DateTimeOffset on persistence layer still, to avoid any problems with DateTime.Kind and time zone conversions

Reply:
check
* https://github.com/swisschain/Sirius.Brokerage/blob/1a409fb83a198279980cb62cf41c1bd93b9e4048/src/Brokerage.Common/Persistence/Accounts/AccountsRepository.cs#L143 
* https://github.com/swisschain/Sirius.Brokerage/blob/1a409fb83a198279980cb62cf41c1bd93b9e4048/src/Brokerage.Common/Persistence/Accounts/AccountsRepository.cs#L166